realm/realm-library/src/main/java/io/realm/internal/RealmObjectProxy.java

```diff
@@ -34,6 +35,7 @@
      * Tuple class for saving meta data about a cached RealmObject.
      */
     class CacheData<E extends RealmModel> {
+        @SuppressFBWarnings("URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD")
         public int minDepth;
```

Comment:
Why silencing FindBug? is this used from the generated class?

Reply:
Yes, it is referenced by our proxy classes. Not entirely sure why it suddenly starts complaining.